script.js

```diff
@@ -1,49 +1,64 @@
-let msgWaiting = document.querySelector(".message-waiting");
-let searchForm = document.getElementById("search-form");
-
+const msgWaiting = document.querySelector(".message-waiting");
+const searchForm = document.getElementById("search-form");
```

Comment:
For consistency, I changed all `let`s to `const`s in the set-up section of the script. A rule of thumb for some is that `const` should be the first choice, using `let` only when we're working with variables that we know are going to change. I'm not personally so strict about this, but I do like consistency. That's because inconsistency makes me ask why – could it be the result of a mistake? Could it cause a bug?